CHANGELOG.md

```diff
@@ -42,6 +42,8 @@
   The default value of the two labels are set to `konghq.com/secret` and
   `konghq.com/configmap`.
   [#1922](https://github.com/Kong/kong-operator/pull/1922)
+- `GatewayConfiguration` `v1beta1` has been replaced by the new API version `v2alpha1`.
```

Comment:
Can we expand this to describe 
- If any user action is required?
- If we're still going to support `v1beta1`? If so are we going to provide a migration webhook?
- Should users first install the new CRD and then upgrade the operator?

Reply:
f9245ba88024de0448fe83d1ec35fc93d882b106